memory.cpp

```diff
@@ -103,7 +103,9 @@
 
     #include <pthread.h>
     #include <assert.h>
-    //#include <execinfo.h>
+    #ifndef USE_MUSL_LIBC
+        #include <execinfo.h>
```

Comment:
Why was this commented out before and now it is not. If USE_MUSL_LIBC is not defined should this code really be included?